.vib/spring-cloud-dataflow/cypress/cypress/integration/prepare-app-state.js

```diff
@@ -0,0 +1,52 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+export const importAStreamApplication = () => {
+  cy.visit('/dashboard');
+  cy.contains('button', 'Add application(s)').click();
+  cy.contains(
+    '.clr-control-label',
```

Comment:
Instead of a class selector (and as we are already using a very specific string `Stream application starters for Kafka/Maven`), could we use a more generic selector here like `button`, which is less prone to change?

Reply:
Changed accordingly.